src/core/writer.ml

```diff
@@ -0,0 +1,22 @@
+(*
+ * Module to write into file
+ * DURAND - MARAIS © 2019
+ *)
+
+
+(* Auxilliary function to write content *)
+let rec writer chan lst = 
+  match lst with
+  | [] -> close_out chan ; Ok true
+  | h::q -> (
+    output_string chan (h^"\n") ;
+    writer chan q
+  )
+
```

Comment:
Could it be a great idea to add a function which totally overwrites the file?

Reply:
This function totally overwrites the file. The selection if we overwrite it or not will be done in the last part (Crown)